app/code/Magento/Review/Model/ResourceModel/Rating/Collection.php

```diff
@@ -314,7 +311,9 @@ protected function _addStoreData()
         if (is_array($data) && count($data) > 0) {
             foreach ($data as $row) {
                 $item = $this->getItemById($row['rating_id']);
-                $item->setStores(array_merge($item->getStores(), [$row['store_id']]));
+                $stores = $item->getStores();
```

Comment:
@fredden This seems Like a good change maybe add it in the desciption of the pull request.